### PR TITLE
[18/10/2022]Task3.2: Added event cards

### PR DESCRIPTION
**Pull request by sandygudie**
### Summary: Issue https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/186
**Tasks 3**
We need to add an events subpage where we can link our events in [Zenodo](https://zenodo.org/communities/1231231664/?page=1&size=20).
3.1 Add this to the nav bar [PR](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/pull/180).
3.2 Add three cards with each event, title, name of the speaker, and language.

### Fixes <NUM>
* Added event cards on the homepage*
* Users can see listed events when they click the `Event` link on the navbar
* The Arabic version of the page is updated 
Result
image

### What should a reviewer concentrate their feedback on?
- [ ] Everything looks ok?

### CODE REVIEW

**Comment by sandygudie** [comment]
@alswajiab @BatoolMM  please review

**Comment by BatoolMM** [comment]
This looks fantastic 🌷

Can you make the link for (More OSCSA events) open a new page on the website with all previous events?

**Comment by sandygudie** [comment]
> This looks fantastic 🌷
> 
> Can you make the link for (More OSCSA events) open a new page on the website with all previous events?

Noted @BatoolMM

**Comment by BatoolMM** [comment]
It looks fantastic, can you move this section to just before the UNESCO recommendation and resolve the conflict, please?

**Comment by BatoolMM** [comment]
@sandygudie can you rebase this PR, please?

File: index.html [hunk 1]
```diff
@@ -365,6 +365,45 @@ <h4>Web 3</h4>
 
       </div>
     </section><!-- End Portfolio Section -->
+     <!-- ======= Event Section ======= -->
+  <section id="event">
+    <div class="container mt-3" data-aos="fade-up">
+      <div class="col-md-12 mb-5">
+        <h3 class="section-title">Events</h3>
```
Comment (BatoolMM) [inline index.html:372]:
```suggestion
        <h3 class="section-title">Materials and Talks</h3>
        <p class="subscribe-text">All our talks, workshops, and training courses by Community lead and OSCSA members in all events and conferences, including 
         <a "https://elifesciences.org ">eLife</a>
          <a href="https://www.unesco.org/en">UNESCO</a>. 
        are under CC-BY license and accessible in
               <a href="https://zenodo.org/communities/1231231664/?page=1&size=20">Zenodo</a>. 
               Any code associated with these materials is included in our <a href="https://github.com/Open-Science-Community-Saudi-Arabia">GitHub
              Organization Account</a>!</p>
```

File: index.html [hunk 2]
```diff
@@ -365,6 +365,45 @@ <h4>Web 3</h4>
...
+            <img  class="card-img-top" src="./assets/img/event/event3.PNG" alt="event-image"/>
+            <div class="card-body">
+              <h6 class="card-title">Demystifying the Command line (Arabic)</h5>
+              <p class="card-text">Speaker: Batool Almarzouq</p>
+              <p class="card-text">Language:Arabic</p>
+            </div>
+          </div>
+        </div>
+        <div class="text-center mt-5">
+        <h6><a  class="font-weight-bold more-event" style="font-size:18px" href="https://zenodo.org/communities/1231231664/?page=1&size=20">  More OSCSA Events</a></h6>
```
Comment (BatoolMM) [inline index.html:402]:
```suggestion
        <h7><a  class="font-weight-bold more-event" style="font-size:18px" href="https://zenodo.org/communities/1231231664/?page=1&size=20">  More OSCSA Materails</a></h7>
```

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Comment by sandygudie** [comment]
> @sandygudie can you rebase this PR, please?

done @BatoolMM

File: ar-index.html [hunk 3]
```diff
@@ -460,7 +460,45 @@ <h3 class="subscribe-title">نشاطاتنا التعليمية</h3>
     </section><!-- End Subscrbe Section -->
 
 
-
+ <!-- ======= Event Section ======= -->
+ <section id="event">
+  <div class="container" data-aos="fade-up">
+    <div class="col-md-12">
+      <h3 class="section-title">الأحداث</h3>
```
Comment (BatoolMM) [inline ar-index.html:467]:
```suggestion
      <h3 class="section-title">نشاطاتنا التعليمية</h3>
            <p class="subscribe-text">جميع مشاركتنا، وورشنا العلمية ودوراتنا التدريبية تحت ترخيص CC-BY و
              متاحة في <a href="https://zenodo.org/communities/1231231664/?page=1&size=20">زينودو (Zenodo)</a>. أي كود مرتبط بهذه المواد مضمن في حسابنا على مؤسسة <a href="https://github.com/Open-Science-Community-Saudi-Arabia">GitHub</a> <a href="https://github.com/Open-Science-Community-Saudi-Arabia"></a>.</p>
          </div>

```

**Comment by sandygudie** [comment]
@BatoolMM 
TypoError , "Materials"
image

File: ar-index.html [hunk 4]
```diff
@@ -460,7 +460,45 @@ <h3 class="subscribe-title">نشاطاتنا التعليمية</h3>
...
+  <div class="container" data-aos="fade-up">
+    <div class="col-md-12">
+      <h3 class="section-title">الأحداث</h3>
+      <div class="section-title-divider"></div>
+    </div>
+      <div class="row gx-0 mb-5" style="grid-gap:2em;">
+        <div class="card col-lg col-md-5 col-sm-1">
+          <img class="card-img-top"src="./assets/img/event/event1.PNG" alt="event-image"/>
+          <div class="card-body">
+            <h6 class="card-title">الاستفادة من العلوم المفتوحة في التعلم الآلي والمعلوماتية الحيوية</h5>
```
Comment (BatoolMM) [inline ar-index.html:474]:
```suggestion
            <h6 class="card-title">الاستفادة من العلوم المفتوحة في الذكاء الإصناعي والمعلوماتية الحيوية</h5>

```

File: ar-index.html [hunk 5]
```diff
@@ -460,7 +460,45 @@ <h3 class="subscribe-title">نشاطاتنا التعليمية</h3>
...
+            <p class="card-text">مكبر الصوت: Batool Almarzouq</p>
```
Comment (BatoolMM) [inline ar-index.html:475]:
```suggestion
            <p class="card-text">د.بتول المرزوق</p>

```

File: ar-index.html [hunk 6]
```diff
@@ -460,7 +460,45 @@ <h3 class="subscribe-title">نشاطاتنا التعليمية</h3>
...
+            <p class="card-text">اللغة الإنجليزية</p>
+          </div>
+        </div>
+        <div class="card col-lg col-md-5 col-sm-1">
+          <img  class="card-img-top"src="./assets/img/event/event2.PNG" alt="event-image"/>
+          <div class="card-body">
+            <h6 class="card-title">نحو العولمة الأوسع للمصدر المفتوح: توثيق رحلة الأقلمة الخاصة بك</h5>
```
Comment (BatoolMM) [inline ar-index.html:482]:
```suggestion
            <h6 class="card-title">نحو عولمة أوسع للمصادر المفتوحة: توثيق رحلة التوطين والترجمة</h5>

```

File: ar-index.html [hunk 7]
```diff
@@ -460,7 +460,45 @@ <h3 class="subscribe-title">نشاطاتنا التعليمية</h3>
...
+            <p class="card-text">مكبر الصوت: Melissa Black and Asma Kacem</p>
```
Comment (BatoolMM) [inline ar-index.html:483]:
```suggestion
            <p class="card-text">اسماء قاسم ومليسا بلاك</p>

```

File: ar-index.html [hunk 8]
```diff
@@ -460,7 +460,45 @@ <h3 class="subscribe-title">نشاطاتنا التعليمية</h3>
...
+            <p class="card-text">اللغة الإنجليزية</p>
+          </div>
+        </div>
+        <div class="card col-lg col-md-5 col-sm-1">
+          <img  class="card-img-top" src="./assets/img/event/event3.PNG" alt="event-image"/>
+          <div class="card-body">
+            <h6 class="card-title">تبسيط سطر الأوامر (عربي)</h5>
+            <p class="card-text">مكبر الصوت: Batool Almarzouq</p>
```
Comment (BatoolMM) [inline ar-index.html:491]:
```suggestion
            <p class="card-text">د.بتول المرزوق</p>

```

File: ar-index.html [hunk 9]
```diff
@@ -460,7 +460,45 @@ <h3 class="subscribe-title">نشاطاتنا التعليمية</h3>
...
+          <div class="card-body">
+            <h6 class="card-title">نحو العولمة الأوسع للمصدر المفتوح: توثيق رحلة الأقلمة الخاصة بك</h5>
+            <p class="card-text">مكبر الصوت: Melissa Black and Asma Kacem</p>
+            <p class="card-text">اللغة الإنجليزية</p>
+          </div>
+        </div>
+        <div class="card col-lg col-md-5 col-sm-1">
+          <img  class="card-img-top" src="./assets/img/event/event3.PNG" alt="event-image"/>
+          <div class="card-body">
+            <h6 class="card-title">تبسيط سطر الأوامر (عربي)</h5>
```
Comment (BatoolMM) [inline ar-index.html:490]:
```suggestion
            <h6 class="card-title">تبسيط لوحة الأوامر السوداء</h5>

```

**Review by BatoolMM — CHANGES_REQUESTED** [review]

File: index.html [hunk 10]
```diff
@@ -366,6 +366,52 @@ <h4>Web 3</h4>
 
       </div>
     </section><!-- End Portfolio Section -->
+     <!-- ======= Event Section ======= -->
+  <section id="event">
+    <div class="container mt-3" data-aos="fade-up">
+      <div class="col-md-12 mb-5">
+        <h3 class="section-title">Materials and Talks</h3>
```
Comment (BatoolMM) [inline index.html:373]:
```suggestion
        <h3 class="section-title"> Materials and Talks</h3>
```

**Comment by BatoolMM** [comment]
> @BatoolMM TypoError , "Materials" image and the divider should be below the title text image

Thank you for catching this!

File: index.html [hunk 11]
```diff
@@ -366,6 +366,52 @@ <h4>Web 3</h4>
...
+            <img  class="card-img-top" src="./assets/img/event/event3.PNG" alt="event-image"/>
+            <div class="card-body">
+              <h6 class="card-title">Demystifying the Command line (Arabic)</h5>
+              <p class="card-text">Speaker: Batool Almarzouq</p>
+              <p class="card-text">Language:Arabic</p>
+            </div>
+          </div>
+        </div>
+        <div class="text-center mt-5">
+        <h7><a  class="font-weight-bold more-event" style="font-size:18px" href="https://zenodo.org/communities/1231231664/?page=1&size=20">  More OSCSA Materails</a></h7>
```
Comment (BatoolMM) [inline index.html:410]:
```suggestion
        <h7><a  class="font-weight-bold more-event" style="font-size:18px" href="https://zenodo.org/communities/1231231664/?page=1&size=20">  More OSCSA Materials and Talks </a></h7>
```

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Comment by sandygudie** [comment]
> > @BatoolMM TypoError , "Materials" image and the divider should be below the title text image
> 
> Thank you for catching this!

i have fixed this

**Comment by BatoolMM** [comment]
I made changes to the Arabic translation - The Materials in the nav bar should direct to the cards you created, can you fix that @sandygudie.

**Comment by sandygudie** [comment]
> I made changes to the Arabic translation - The Materials in the nav bar should direct to the cards you created, can you fix that @sandygudie.

Alright, I will fix that right away

**Comment by BatoolMM** [comment]
Thank you @sandygudie 🌷 , for the speedy fix, I noticed that the Arabic page has no padding in all directions - the card reaches the end of the screen.

Screenshot 2022-10-25 at 22 35 12

In terms of the English page, needs padding on the top and bottom only.

Screenshot 2022-10-25 at 22 35 43

Can you fix this issue, I'd love to merge this today but no rush!

**Comment by sandygudie** [comment]
> Thank you @sandygudie 🌷 , for the speedy fix, I noticed that the Arabic page has no padding in all directions - the card reaches the end of the screen.
> 
> Screenshot 2022-10-25 at 22 35 12
> 
> In terms of the English page, needs padding on the top and bottom only.
> 
> Screenshot 2022-10-25 at 22 35 43
> 
> Can you fix this issue, I'd love to merge this today but no rush!

I will fix that now

**Comment by BatoolMM** [comment]
Fantastic! 
Isn't the card supposed to direct you to its page in Zenodo once it's clicked but that's not the case?

**Comment by sandygudie** [comment]
> Fantastic @sandygudie !
> Isn't the card supposed to direct you to its page in Zenodo once it's clicked but that's not the case?

Based on the Task description(Task 3) clicking on a card should open a more detailed subpage  with description ,  to view slides or download etc

https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/186

**Comment by BatoolMM** [comment]
@sandygudie It would be nice for each card would direct you to a subpage - but this can be made in a separate PR - otherwise, this will be heavy PR. To merge this PR, it should at least direct you to the slides from Zenodo (their page in Zenodo), if that makes sense! Let me know if that's not clear!

**Comment by RealRichi3** [comment]
@sandygudie if you need assistance with the subpages, i'm open to assist

**Comment by sandygudie** [comment]
> 

Yes, that’s right @BatoolMM,

The subpage is been worked on here https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/pull/191

**Review by BatoolMM — APPROVED** [review]

**Comment by BatoolMM** [comment]
Looks fantastic, thank you again @sandygudie!

**Comment by BatoolMM** [comment]
@sandygudie, the CSS for the Arabic page is completely distorted - can you open a PR to fix it, please?

Screenshot 2022-10-26 at 08 20 24

**Comment by sandygudie** [comment]
Oh No, i had  just pushed a fixed  before you merged, I will fix it quick
thanks

**Comment by sandygudie** [comment]
@BatoolMM the fix : https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/pull/214

**Pull request merged**